src/wp-includes/comment-template.php

```diff
@@ -2446,6 +2446,7 @@ function wp_list_comments( $args = array(), $comments = null ) {
  * @since 4.6.0 Introduced the 'action' argument.
  * @since 4.9.6 Introduced the 'cookies' default comment field.
  * @since 5.5.0 Introduced the 'class_container' argument.
+ * @since n.e.x.t Introduced the 'novalidate_form' argument.
```

Comment:
I used this placeholder version since I wasn't sure how long it would take to merge this or if we'd target it for 6.8.2. But seems like it makes sense for a minor release, especially with the current relaxation of the criteria.

Reply:
I agree; 6.8.2 seems reasonable. But with all the changes recently, I honestly don't have any idea...